NamingSystem/NamingSystem-Genomics-report-number.json

```diff
@@ -0,0 +1,43 @@
+{
+  "resourceType": "NamingSystem",
+  "id": "genomics-report-number",
+  "name": "GenomicsReportNumber",
+  "status": "draft",
+  "kind": "identifier",
+  "date": "2025-02-10",
```

Comment:
Do we want to use a recent date? Cant remember if we haven`t had IG release since February, and this assets is newly going into an IG version.